source/command-monitoring/tests/unacknowledgedBulkWrite.yml

```diff
@@ -12,9 +12,10 @@ tests:
       name: "bulkWrite"
       arguments:
         requests:
-          - insertOne: 
+          - name: "insertOne"
+            arguments:
               document: { _id: "unorderedBulkWriteInsertW0", x: 44 }
-        ordered: false
+        options: { ordered: false }
         writeConcern: { w: 0 }
```

Comment:
I find this confusing about the other test runner as well, but I suspect its because of python kwargs. If we are introducing an `options` field, it stands to reason that we would put all of our options there. Why do we special case some fields like `writeConcern`?

Reply:
I believe this is because `ordered` is always an option when creating a bulk write object, and write concern is either inherited from the parent collection or specified per-option (in some drivers) at the time the bulk write is executed. Placing these together in `options` would require the test harness to pull out certain options.

Reply:
Right: at least in PyMongo, this gets implemented like:

```
wc = ... get writeConcern from test ...
options = ... get options like "ordered" from test ...
collection = collection.with_options(write_concern=wc)
collection.bulk_write(requests, options)
```
This is consistent with the CRUD Spec, which puts "ordered" in the options passed to bulk_write(), but doesn't put writeConcern there.

Anyway, my goal here is a small step towards consistency among test runners. If you want, I can cancel this in favor of one big change later this summer that unifies them all.